Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -657,11 +661,12 @@ buildVehicle req personId merchantId = do
         SV.updatedAt = now
       }
 
-makeDriverInformationRes :: DriverEntityRes -> DM.Merchant -> Maybe (Id DR.DriverReferral) -> DriverInformationRes
-makeDriverInformationRes DriverEntityRes {..} org referralCode =
+makeDriverInformationRes :: DriverEntityRes -> DM.Merchant -> Maybe (Id DR.DriverReferral) -> DriverStats -> DriverInformationRes
+makeDriverInformationRes DriverEntityRes {..} org referralCode driverStats =
   DriverInformationRes
     { organization = DM.makeMerchantAPIEntity org,
       referralCode = referralCode <&> (.getId),
+      numberOfRides = driverStats.totalRides,
```

Comment:
if you want total rides as in (cancelled + completed + inprogress...) then use `totalRidesAssigned`, `totalRides` is only completed rides.

Reply:
@piyushKumar-1  we need only completed rides